tests/lib/nested.sh

```diff
@@ -1230,6 +1229,48 @@ nested_exec_as() {
     sshpass -p "$PASSWD" ssh -p "$NESTED_SSH_PORT" -o ConnectTimeout=10 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "$USER"@localhost "$@"
 }
 
+nested_prepare_tools() {
+    TOOLS_PATH=/writable/test-tools
```

Comment:
Hm spread runs seem to be unhappy with this path. For classic systems there's permission denied creating this path, which is expected I suppose. We should be using /usr/bin, or better /usr/local/bin and first make sure that it's in the standard path for the test user.

However, I'm not sure why we see `mkdir: cannot create directory ‘/writable/test-tools’: Permission denied`, is /writable remounted as read only?

Reply:
fixed